nomad/worker.go

```diff
@@ -254,6 +254,7 @@ func (w *Worker) invokeScheduler(snap *state.StateSnapshot, eval *structs.Evalua
 	}
 
 	// Create the scheduler, or use the special system scheduler
+	// QUESTION: does this mean "special core scheduler"
```

Comment:
Yes! this is `special core scheduler` that runs some administrative tasks (e.g. gc).